content-src/lib/filters.js

```diff
@@ -16,7 +13,7 @@ function createFilter(definition, name) {
     let result = true;
     definition.forEach((test, i) => {
       if (!test(item)) {
-        log(`omitting ${item.url}\nbecause it failed test #${i} of ${name}`);
+        console.log(`omitting ${item.url}\nbecause it failed test #${i} of ${name}`);
```

Comment:
We may want to add back the `const {log} = require("lib/logger");` bit so we aren't always sending the output to the user's DevTools console:

```
/home/travis/build/mozilla/activity-streams/content-src/lib/filters.js
  16:9  warning  Unexpected console statement  no-console
```